Python driver emits ERROR log: Cloud not read version file

#### Description
K8s controller emits error log:
2017/12/19 17:22:41 [WARNING] Overwriting existing entry for backend {ServiceName:svc ServicePort:80 Namespace:default}
2017/12/19 17:22:41 [INFO] Wrote 1 Virtual Server and 0 IApp configs
**2017/12/19 17:22:46 [ERROR] [2017-12-19 17:22:45,946 __main__ ERROR] Could not read version file: [Errno 2] No such file or directory: 'VERSION_BUILD.json'**
2017/12/19 17:22:48 [INFO] [2017-12-19 17:22:48,541 __main__ INFO] entering inotify loop to watch /tmp/k8s-bigip-ctlr.config307573180/config.json
2017/12/19 17:22:50 [INFO] [2017-12-19 17:22:49,948 f5_cccl.resource.resource INFO] Creating ApiHTTPMonitor: /test/ingress_default_svc_0_http

Please verify this issue does not also effect marathon-bigip-ctlr, and cf-bigip-ctlr.

#### Kubernetes Version
v1.7.5

#### Controller Version
Version: 1.4.0-WIP, BuildInfo: ubuntu-207bd5a-dirty-20171218230526

#### BIG-IP Version
12.1

#### Diagnostic Information

--- a/python/bigipconfigdriver.py
+++ b/python/bigipconfigdriver.py
@@ -697,7 +697,7 @@ def _handle_vxlan_config(config):
 
 def _set_user_agent():
     try:
-        with open('VERSION_BUILD.json', 'r') as version_file:
+        with open('/app/python/VERSION_BUILD.json', 'r') as version_file:
             data = json.load(version_file)
             user_agent = \
                 "k8s-bigip-ctlr-" + data['version'] + '-' + data['build']

